tests/integrationv2/test_compatibility_with_oqs_openssl.py

```diff
@@ -0,0 +1,272 @@
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2019_06,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2019_06,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2020_02,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2020_02,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.PQ_SIKE_TEST_TLS_1_0_2019_11,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.PQ_SIKE_TEST_TLS_1_0_2019_11,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.PQ_SIKE_TEST_TLS_1_0_2020_02,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.PQ_SIKE_TEST_TLS_1_0_2020_02,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    # The last set of vectors verify that a standard handshake can be completed when only one side support PQ
+    # by specifing a "mismatch" between PQ cipher preferences
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2019_06,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_PQ_TLS_1_0_2020_02,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_TLS_1_0_2018_10,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+    {
+        "client_ciphers": Ciphers.KMS_TLS_1_0_2018_10,
+        "server_ciphers": Ciphers.ECDHE_RSA_AES256_GCM_SHA384,
+        "expected_cipher": "ECDHE-RSA-AES256-GCM-SHA384",
+        "expected_kem": "NONE",
+    },
+]
```

Comment:
I think you can remove (almost) all of these test vectors. OQS_OpenSSL doesn't supported PQ 1.2 (as far as I know), and we don't have PQ for TLS 1.3 in s2n yet, so these aren't really testing anything. We'll have to add our own test vectors once create cipher suites/KEM preferences/security policies once we start the client/server pq 1.3 implementation.

For now, I think we really only need one vector just to prove that we are building oqs_openssl correctly and s2n is communicating with it as expected. Suggest making the test vector use TLS 1.3 ciphers for both s2n and oqs so that we can confirm we are negotiating a 1.3 handshake. (And that one test vector is really just a placeholder anyway - will be replaced once we have pq 1.3 cipher suites in s2n.)